R/dummy_functions.R

```diff
@@ -95,20 +95,27 @@ example_module <- function(label = "example teal module", datanames = "all") {
     server = function(id, data) {
       checkmate::assert_class(data(), "teal_data")
       moduleServer(id, function(input, output, session) {
-        output$text <- renderPrint(data()[[input$dataname]])
+        ns <- session$ns
+        output$datanames <- renderUI({
+          selectInput(ns("dataname"), "Choose a dataset", choices = teal.data::datanames(data()))
+        })
```

Comment:
Each time data is filtered renderUI renders input. This means that when you set selection to `"ADTTE"`, after filtering this will change back to `"ADSL"`
```suggestion
        isolate({
          selectInput(ns("dataname"), "Choose a dataset", choices = teal.data::datanames(data()))
        })
```

Reply:
You mean to use `updateSeletInput` here, correct?

Reply:
yes